.gitignore

```diff
@@ -12,5 +12,13 @@ public/images/avatars/*
 /config/admin.yml
 /config/config.yml
 /config/database.yml
+/config/tweets-client.ini
+/config/secrets.yml
+/config/skylight.yml
+/*~
+
 .*.swp
 .DS_Store
+*~
+\#*\#
+.\#.*
```

Comment:
@DavidLemayian backup files created by editors. 

As for the avatars, yes we are ignoring. In production, they are pushed to s3 and the 3,702 avatars in public/ and app/assets were those of U.S politicians.